spec/models/room_spec.rb

```diff
@@ -3,5 +3,7 @@
 RSpec.describe Room, type: :model do
   describe 'relationships' do
     it {should belong_to :hotel}
+    it {should have_many :guest_rooms}
+    it {should have_many(:guests).through(:guest_rooms)}
   end
 end
```

Comment:
This test file would be where I would expect a `#guest_count` instance method test.